pkg/util/ipvs/testing/fake.go

```diff
@@ -149,6 +149,16 @@ func (f *FakeIPVS) GetRealServers(serv *utilipvs.VirtualServer) ([]*utilipvs.Rea
 	return f.Destinations[key], nil
 }
 
+func toRealServerNum(dests []*utilipvs.RealServer, destToDel *utilipvs.RealServer) int {
```

Comment:
This function seem wired. Why not just return the RealServerKey as `toVirtualServerKey()` does?